python/ray/util/client/worker.py

```diff
@@ -97,16 +97,41 @@ def __init__(
         self.server = None
         self._conn_state = grpc.ChannelConnectivity.IDLE
         self._converted: Dict[str, ClientStub] = {}
+        self._secure = secure
+        self._conn_str = conn_str
+        self._connection_retries = connection_retries
 
         if secure and _credentials is None:
-            _credentials = grpc.ssl_channel_credentials()
+            self._credentials = grpc.ssl_channel_credentials()
```

Comment:
Will this be reused during retries? Can they expire?

Reply:
Hmm, that is a really good question for the user-provided credentials--I think they have some built in expiration time. 

This being said, I think that it is usually a time-based expiration (this token expires at time X, as opposed to "this token cannot be reused").

Reply:
My preference would be to keep credentials as a local variable as before. Then there wouldn't be any expiration issue.

Otherwise we should leave a comment on why storing the expirable self._credentials is ok when it is reused after expiration, and necessary.